### PR TITLE
Minh

### PR DESCRIPTION
**Pull request by minhbui27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ Team *xos*:
 
 ## Development style guide
 
-- Use tabs for indentation as opposed to spaces
+- Minh Bui
```
Comment (neilbalch) [inline README.md]: I think this was a mistake, right? :)

File: TicTacToe.hpp [hunk 2]
```diff
@@ -22,16 +22,36 @@ class TicTacToe
...
+			blue_points = checkVictoryHorizontal(blue) + checkVictoryDiagonal(blue) + checkVictoryVertical(blue);
+			red_points = checkVictoryHorizontal(red) + checkVictoryDiagonal(red) + checkVictoryVertical(red);
+			if(blue_points == 4) {
+				victory = 1;
+			}
+			else if(red_points = 4) {
+				victory = -1;
+			}
+			else {
+				blue > red ? victory = 1 : victory = -1;
```
Comment (neilbalch) [inline TicTacToe.hpp]: We don't need this case, because we only care if someone has won, not if they are in the lead.

Comment (minhbui27) [reply TicTacToe.hpp]: ok

File: README.md (hunk 1 above)
Comment (minhbui27) [reply README.md]: the file probably automatically did that, sorry

File: TicTacToe.hpp [hunk 3]
```diff
@@ -22,16 +22,33 @@ class TicTacToe
...
+		} // Minh
 		int getVictory() {
 			return victory;
 		}
 
 	private:
 		// These functions should only be called from within this class.
 		void changeTurn(); //Phillip
-		void checkVictory();
+		void checkVictory() {
```
Comment (neilbalch) [inline TicTacToe.hpp]: Oh, wait! Please implement this in `TicTacToe.cpp`, not here.

**Review by hanzemeng — APPROVED** [review]

**Pull request merged**